tests/logictest/suites/base/02_function/02_0012_function_datetimes

```diff
@@ -800,7 +800,23 @@ select to_year(to_datetime(1646404329000000)) = 2022;
 ----
 1
 
+statement query T
+select date_add(QUARTER, 1, to_date('2018-01-02'));
+
+----
+2018-04-02
+
+statement query T
+select date_sub(QUARTER, 1, to_date('2018-01-02'));
 
+----
+2017-10-02
+
+statement query T
+select date_trunc(month, to_date('2022-07-07'));
```

Comment:
`QUARTER` ?

Reply:
This statement is just for `date_trunc`，so `MONTH` is fine.